tests/lib/pages/projects_versions_page.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'base_page'
+class ProjectsVersionsPage<BasePage
+
+ link(:projects_versions,css:"div #tab-content-versions p a.icon-add")
```

Comment:
too many nodes in the locator value
there is no any specific rule about it
but less is better